### ISSUE TITLE
Feature request: less verbose notifications & manuel sync

### ISSUE THREAD

**Issue by obriat** [aleskandro/nextcloud-kobo#1]
Hi,

Thanks for this pluginin, it works perfectly on the Kobo Libra Color.

I just have a couple of suggestions:

- The notifications are to numerous, is it possible to keep it to one general popup? Ex: "Doc1, doc2 doc3, and X other docs have been updated"
- Is possible to start update only on Kobo manual sync (the top circle arrows button)?  If it's the behaviour of `autoUpdat: false`, can you add this info to the option description?

**Comment by aleskandro** [comment]
Hi @obriat I'll send a patch soon for reducing the number of notifications. I've been trying having it more reliable in cases where the network is not super reliable and needed more verbose notifications to support that.

Apologize for the late reply, I have lost this notification and have been busy on other projects.

Aleskandro

**Comment by aleskandro** [comment]
Also, I'd open another issue for the second point you reported.

**Comment by obriat** [comment]
Thanks.
I just find out that `autoUpdate` means application autoupdate (and not doc updates) 😁. Don't know if I'm the only one, but this option description in the readme could use application instead of daemon?
By the way what's the way to manual update it?

**Comment by aleskandro** [comment]
If autoUpdate is not an option for you, you can drop the KoboRoot.tgz in the device just as you did during the first installation.

**Comment by obriat** [comment]
No, it's just "for  the record". Another question : how I can know the app version from the Kobo?

**Comment by aleskandro** [comment]
The version can be found at /mnt/onboard/.adds/nextcloud-kobo/version.txt.

Also, the verbosity has been reduced.

For manual sync, we'd open another issue.
